contents/blog/session-replay-pricing.md

```diff
@@ -0,0 +1,94 @@
+---
+date: 2024-07-22
+title: "We’ve decided to make less money [Part 1]"
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/take_my_money_11433edb48.png
+author:
+  - andy-vandervell
+featuredImageType: full
+tags:
+  - PostHog news
+---
+
+## TL;DR
+ 
+- We’ve optimized our session replay pipeline and infrastructure.
+
+- We’re passing the savings onto you by making session replay _drastically cheaper_, especially at lower volumes.
+
+- We’re now the cheapest session replay tool among competitive products.
+
+And this is just Part 1...
+
+## How will this impact me?
+
+It depends on usage, but a customer generating 25k recorded sessions a month will now pay $85 a month, down from $430 a month. That's a huge $4,140 annual saving.
+
+This makes using PostHog cheaper than any competing tool with similar features that publishes its pricing.
+
+![new pricing](https://res.cloudinary.com/dmukukwp6/image/upload/new_pricing_7e0af9ddc4.jpg)
+
+We’re not just cheaper than replay tools designed for developers, either. Anyone using Hotjar for their website or web app can also save by switching to PostHog.
+
+And, unlike Hotjar, we support [session replays on mobile](/docs/session-replay/mobile) with our open beta for [iOS](/docs/libraries/ios) and [Android](/docs/libraries/android) SDKs – yes, [we are working on React Native](https://github.com/PostHog/posthog/issues/13269), folks, but feel free to vote for it on [our public roadmap](/roadmap) anyway.
+
+![posthog-vs-hotjar](https://res.cloudinary.com/dmukukwp6/image/upload/hotjar_ace01c5cb7.jpg)
+
+Sadly, many tools don’t publish pricing. That’s their choice, but we’re pretty confident it’s not because they’re secretly great value. How else are they going to justify those huge outbound sales teams, eh?
+
+You can [calculate your new price](/pricing) on the pricing page, but here’s a breakdown of the changes at different volumes:
+
+| &nbsp;                 | **Old price** | **New price** | **Change**   |
+|------------------------|---------------|---------------|--------------|
+| **5,000 recordings**   | $0            | $0            | No change    |
+| **25,000 recordings**  | $430          | $85           | 5.1x cheaper |
+| **50,000 recordings**  | $505          | $173          | 2.1x cheaper |
+| **100,000 recordings** | $640          | $273          | 2.3x cheaper |
+| **250,000 recordings** | $1,025        | $543          | 1.9x cheaper |
+
+As before, every user gets 5,000 free recordings every month, too. Free stuff isn’t limited to the free tier.
+
+## Why are you doing this?
```

Comment:
It feels like all these titles are framed differently to the body copy and it's unclear. 

In the body copy, 'you' = the user and 'we' = PostHog. But it's the opposite in the titles. It also seems a little strange to position it this way around. 

I'd just change all the titles. So this one becomes

```suggestion
## Why are we doing this?
```